### PR TITLE
[BMB-22] Refactoring actions

### PR DESCRIPTION
**Pull request by gollariel**

### CODE REVIEW

File: containers/bamba-core/bamba/errors.go [hunk 1]
```diff
@@ -0,0 +1,9 @@
+package bamba
+
+import "errors"
+
+// All kind of error for bamba
+var (
+	ErrInvalidType  = errors.New("can not be casted to given type")
+	ErrUnknowAction = errors.New("unknown action")
```
Comment (thepeak99) [inline containers/bamba-core/bamba/errors.go]: Unknown ;-)

Comment (gollariel) [reply containers/bamba-core/bamba/errors.go]: Hm... I not sure what do you mea?n :)

Comment (thepeak99) [reply containers/bamba-core/bamba/errors.go]: hehe I mean `ErrUnknowAction` should be `ErrUnknownAction`

Comment (gollariel) [reply containers/bamba-core/bamba/errors.go]: Oh. Yes. Sorry. You are right! I looked at the message, and do not check variable

**Comment by thepeak99** [comment]
@gollariel I'm merging this one so it's easier to review the next one :laughing:

**Pull request merged**